maro/rl/models/representation_layers.py

```diff
@@ -4,7 +4,7 @@
 import torch.nn as nn
 
 
-class MLPRepresentation(nn.Module):
+class RepresentationLayers(nn.Module):
     def __init__(self, name: str, input_dim: int, hidden_dims: [int], output_dim: int, dropout_p: float):
         """Deep Q network.
```

Comment:
I see much duplicate code in decision layers and representation layers, resolve them